src/scripts/install_aws_iam_authenticator.sh

```diff
@@ -1,31 +1,34 @@
 #!/bin/bash
+set -x
```

Comment:
Was leaving the `set -x` here intentional? It will output a lot of stuff that is not necessarily relevant to users.